src/main/webapp/search.js

```diff
@@ -78,14 +78,14 @@ function addVideoInfo(video) {
   cardBodyDiv.className = 'card-body';
   cardBodyDiv.innerHTML = '<p class="card-text">' +
     video.snippet.description + '</p';
-  cardDiv.innerHTML = '<IMG class="card-img-top" src=' +
-    video.snippet.thumbnails.high.url + ' alt="Card image cap">';
+  cardDiv.innerHTML = '<iframe width="480" height="315"src=' + 'https://www.youtube.com/embed/'+video.id.videoId+ '?controls=0;' +
```

Comment:
Why disable video player controls? Does this mean that users won't be able to pause/forward the video?

Also should we set `disablekb` to `1`? This will prevent keyboard inputs from affecting the player.